build_projects/shared-build-targets-utils/Utils/BuildVersion.cs

```diff
@@ -7,7 +7,7 @@ public class BuildVersion : Version
         public string SimpleVersion => $"{Major}.{Minor}.{Patch}.{CommitCountString}";
         public string VersionSuffix => $"{ReleaseSuffix}-{CommitCountString}";
         public string NuGetVersion => $"{Major}.{Minor}.{Patch}-{VersionSuffix}";
```

Comment:
Appears to be used here: https://github.com/dotnet/core-setup/blob/da800066269351dddae310dc262e2184c674d790/build_projects/shared-build-targets-utils/Utils/Utils.cs#L150 which I seems to never be called.